### DISCUSSION TITLE
How to use IPv6? چگونه از آی پی ۶ استفاده کنیم؟

### DISCUSSION THREAD

**Discussion by hiddify-com** [hiddify/Hiddify-Manager#148] [Q&A سوالات رایج] [answered]
بعضی از سرور ها IPv6 ارائه میدند چه جوری تو هیدیفای ازش استفاده کنیم؟

**Comment by hiddify-com** [comment, marked as answer]
کافی است در قسمت dns آی پی ورژن ۶ خود را به عنوان رکورد aaaa استفاده کنید

**Reply by raminrez** [reply]
من اینکار رو روس DNS کلادفلر کردم ولی با ارور آی‌پی ۴ مواجه میشم

<img width="1237" alt="Screenshot 2023-02-15 at 1 32 39 PM" src="https://user-images.githubusercontent.com/7345325/218997610-b4ac4240-fb9c-4096-914e-05575ceb3ea5.png">

**Reply by ghost** [reply]
وقتی مود رو دایرکت انتخاب می کنید یعنی باید حتما تیک پروکسی رو برداشته باشید از cloudflare تا  ip سرور رو برداره

**Reply by raminrez** [reply]
این کار رو انجام دادم
ولی همچنان ارور دارم

<img width="1254" alt="Screenshot 2023-02-15 at 2 24 18 PM" src="https://user-images.githubusercontent.com/7345325/219008400-31acaf9d-24a6-4eea-8299-100bdaf2ed0c.png">
<img width="1111" alt="Screenshot 2023-02-15 at 2 24 50 PM" src="https://user-images.githubusercontent.com/7345325/219008435-3f44e56f-a8ac-4f89-a875-10eeb24d758d.png">

**Reply by ghost** [reply]
از ip6 به عنوان cdn استفاده کنید که translate می شه به 4 با proxy
دایرکت ipv4 باشه اون هم بدون proxy که ip اصلی سرور بتونه ارئه بده

**Reply by raminrez** [reply]
به درسته٬ آیا امکان این هست که سیستم با IP6 کار کنه فقط و نخایم پروکسی کنیم؟ احتمال کثیف بودن این آی‌پی ها کمتره و پیدا کردن تمیزش هم راحت تره

**Reply by ghost** [reply]
تا اونجا که اطلاع دارم واسه وارد شدن به پنل یه ip4 لازم داری دایرکت باشه واسه امنیت این Ip هم حداقل یه دامین , شما توجه کنی اون ip6 که با پروکسی cdn کردی توسط cloudflare ترنسلیت می شه به 4

**Reply by hiddify-com** [reply]
امکانش هست که مستقیم وارد پنل شد با ipv6
کافیه ipv6 را به عنوان دامنه relay ثبت کنید

**Reply by ghost** [reply]
@hiddify
آیا بدون پروکسی باید آی پی 6 رو ریلی باید کرد؟؟؟
 اگر جواب مثبته ریلی کردنی با اینکه اروری نمی ده و قبول می کنه پنل با اون بالا نمیاد و قسمت آی پی جلوی ریلیی که ساختیم یه کاتنتی عجیب غریب می نویسه که توش آیپی 6من هم هست, تو پروکسی های ایجاد شده با دامین اون آِ پی 6 ریلیی چیزی پینگ نمیشه
برای مثال اینطوری (مقادیر الکیه)

 8a88:8ff:f8:a888::1    برای کلود ای پی 6 داده شده
![ip6](https://user-images.githubusercontent.com/125456733/219953723-c77c310c-ad0a-471a-a7e3-a0e7da4f678f.jpg)

**Reply by mohammadi939** [reply]
منم همین مشکل رو دارم
